client/src/shared/Constants.ts

```diff
@@ -1,4 +1,5 @@
 import { OptionTypeBase } from 'react-select';
+import json_schema from './acl_filter.json'
```

Comment:
conventionally, we don't use snake_case in javascript, maybe just rename this to `FilterJsonSchema` and `FilterSchema` below as well?